presto-hive/src/main/java/io/prestosql/plugin/hive/HiveTableProperties.java

```diff
@@ -210,6 +216,18 @@ public static Double getOrcBloomFilterFpp(Map<String, Object> tableProperties)
         return (Double) tableProperties.get(ORC_BLOOM_FILTER_FPP);
     }
 
+    public static Optional<Character> getCsvProperty(Map<String, Object> tableProperties, String csvPropertyKey)
```

Comment:
Nit: maybe shorten to `propertyKey` or just `key` since the full name is redundant given the method name